viewer/static/js/app/components/views/chart.js

```diff
@@ -88,13 +88,18 @@ export default class extends React.Component {
                 axisType="x"
                 refLabels={props.refLabels}
                 size={props.size.innerHeight}
+                height={props.size.height}
+                width={props.size.width}
+                xunit={props.xunit}
               />
               <ChartAxisContainer
                 metricId={props.metricId}
                 scale={props.yScale}
                 axisType="y"
                 refLabels={props.refLabels}
                 size={props.size.innerWidth}
```

Comment:
We should't have done this...think it was me. `size` should be consistent. Just an observation unrelated to this commit.